src/components/studies/participantHooks.ts

```diff
@@ -0,0 +1,90 @@
+import { useUserSessionDataState } from "@helpers/AuthContext"
+import ParticipantUtility, {ParticipantData} from "./participants/participantUtility"
+import { ExtendedError, ExtendedParticipantAccountSummary, ParticipantAccountSummary, ParticipantActivityType, ParticipantEvent } from "@typedefs/types"
+import { useMutation, useQuery, useQueryClient } from "react-query"
+import ParticipantService from "@services/participants.service"
+import EventService from "@services/event.service"
+
+export const PARTICIPANT_KEYS = {
+    all: ['participants'] as const,
+    list: (studyId: string | undefined, currentPage: number = 0, pageSize: number = 0, tab: ParticipantActivityType = 'ACTIVE', searchValue = undefined) =>
```

Comment:
I don't think ParticipantActivityType should be optional. The reason is that when we invalidate this query we want to  make sure that the right one gets invalidated. e.g. if we withdraw someone we want to make sure that we make the 'WITHDRAWN' query stale